rust/src/srtp_parameters.rs

```diff
@@ -35,3 +53,32 @@ impl Default for SrtpCryptoSuite {
         Self::AesCm128HmacSha180
     }
 }
+
+impl ToString for SrtpCryptoSuite {
```

Comment:
This will be removed on the next PR by using `SrtpCryptoSuite` type in flatbuffers.

Reply:
These are great comments, but they really should be TODOs right in the code